mreg/api/permissions.py

```diff
@@ -130,13 +130,13 @@ def has_perm(self, hostname, ips):
     def has_obj_perm(self, obj):
         return self.has_perm(self._get_hostname_and_ips(obj))
 
-    def is_super_or_admin(self, request):
+    def is_super_or_admin(self):
```

Comment:
Hva med å heller implementere en
```
is_super_or_admin(user):
    return user_is_superuser(user) or user_is_adminuser(user)
```
Det virker for meg mer naturlig å teste brukerobjektet i disse tilfellene?